modules/govuk/manifests/apps/link_checker_api.pp

```diff
@@ -64,7 +64,7 @@
   $sentry_dsn = undef,
   $google_api_key = undef,
   $port = 3208,
-  $redis_host = undef,
+  $redis_host = 'redis-2.backend',
```

Comment:
I think this is going to get overwritten by https://github.com/alphagov/govuk-puppet/blob/21f35fcafe1727c7c67840ffd7297436124dbaff/hieradata/common.yaml#L288.  

I'd change it in the common.yaml anyway as it makes it easier to see what's pointing where.